jumpscale/packages/vdc_dashboard/frontend/components/vdcSettings/Backups.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <div>
+    <div class="actions mb-3">
+      <h1 class="d-inline" color="primary" text>Backup &amp; Restore</h1>
+      <v-btn
+        class="float-right p-4"
+        color="primary"
+        text
+        @click.stop="createBackup"
+      >
+        <v-icon left>mdi-plus</v-icon>Create backup
+      </v-btn>
+    </div>
+    <v-data-table
+      :loading="loading"
+      :headers="headers"
+      :items="backups"
+      class="elevation-1"
+    >
+      <template slot="no-data">No VDC instances available</template>
+      <template v-slot:item.name="{ item }">
+        <div>{{ item.name }}</div>
+      </template>
+
+      <template v-slot:item.start_timestamp="{ item }">
+        <div>{{ timeDifference(item.start_timestamp) }}</div>
+      </template>
+
+      <template v-slot:item.expiration="{ item }">
+        <div>{{ timeDifference(item.expiration) }}</div>
+      </template>
+
+      <template v-slot:item.status="{ item }">
+        <v-chip :color="getStatus(item.status)" dark>{{ item.status }}</v-chip>
+      </template>
+
+      <template v-slot:item.actions="{ item }">
+        <v-tooltip top>
+          <template v-slot:activator="{ on, attrs }">
+            <v-btn icon @click.stop="restore(item.name)">
```

Comment:
we can create a confirmation dialog with the restore too because it can be harmful, with a warning that it will restore the system to the state of the backup, any unsaved changes will be lost